doc/api/fs.md

```diff
@@ -1608,19 +1608,19 @@ On Linux, positional writes don't work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
-## fs.write(fd, data[, position[, encoding]], callback)
+## fs.write(fd, string[, position[, encoding]], callback)
```

Comment:
Correct! It does seem a bit convoluted, yes. This time around I only meant to fix the bugs and make sure the documentation is correct, though :)